doc/howtos/using_ubuntu_as_sos.rst

```diff
@@ -0,0 +1,184 @@
+
+Step #2,  setup proxy with you prefered ways, command line tool or system settings tool.  If you don’t need to setup proxy, you could 
+skip this step
+
+Step #3,  installing ssh service with below commands and allow “root” login over SSH optional, 
+People could just ssh into the Service OS and use 'sudo' once logged in (which would be preferable from a security standpoint), then 
+restart SSH service. Of cause, this step is not must 
+if you know how to play, just skip this step.
+# sudo apt-get install openssh-server
+# sudo service ssh status
+In /etc/ssh/sshd_config 
+Change PermitRootLogin from prohibit-password to yes
+# sudo service ssh restart
+
+Step #4, additional packages needed to install, please check the related sections of getting started guide in the ACRN open source
+project, if you don’t have an chance to setup additional packages flowing up that guid to build source. You can reference below commands
+to install directly here. 
+# sudo apt-get update 
+# sudo apt-get install git
+# sudo apt-get install libpciaccess-dev
+# sudo apt-get install gnu-efi
+# sudo apt-get install uuid-dev
+# sudo apt-get install libssl-dev
+# sudo apt-get install vim
+# sudo apt-get install libsystemd-dev
+# sudo apt-get install libevent-dev
+# sudo apt-get install libxml2-dev
+
+Step #5, Checkout source code from open source project repositories
+# git clone https://github.com/projectacrn/acrn-hypervisor
+Note: if you encounter issues with “git clone”, you can download ZIP package directly. It’s so easy.
+
+Step # 6, building the ACRN hypervisor and device model source code on Ubuntu system
+Building device model and hypervisor from source trees on NUC you just setup. You can reference the section of Getting started guide 
+of the ACRN open source project, or following up below commands to build directly here. 
+# cd acrn-hypervisor
+# make PLATFORM=uefi
+The build results are found in the build directory.
+For example:
+# ls acrn-hypervisor/build
+devicemodel
+hypervisor
+tools
+
+Steps #7,  Preparing Service OS kernel 
+You can download latest Service OS kernel from below links:
+https://download.clearlinux.org/releases/current/clear/x86_64/os/Packages/
+
+for example:
+https://download.clearlinux.org/releases/current/clear/x86_64/os/Packages/linux-pk414-sos-4.14.41-39.x86_64.rpm
+or
+https://cdn.download.clearlinux.org/current/x86_64/os/Packages/linux-pk414-sos-4.14.41-39.x86_64.rpm
+
+# mkdir ~/kernel-build
+# cd ~/kernel-build
+# wget  https://download.clearlinux.org/releases/current/clear/x86_64/os/Packages/linux-pk414-sos-4.14.41-39.x86_64.rpm
+# sudo apt-get install rpm2cpio
+# rpm2cpio linux-pk414-sos-4.14.41-39.x86_64.rpm | cpio -idmv 
+# sudo cp -r ~/kernel-build/usr/lib/modules/4.14.41-39.pk414-sos/ /lib/modules/
+
+Step #8, Deploying the hypervisor
+Mount the EFI partition and verify you have the following files, then add the ACRN hypervisor and Clear Linux kernel to the EFI 
```

Comment:
We should refer to this kernel as the Service OS kernel and not the Clear Linux kernel.